storage/src/vespa/storage/common/bucket_utils.h

```diff
@@ -0,0 +1,20 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <vespa/document/bucket/bucketid.h>
+#include <vespa/persistence/spi/bucket_limits.h>
+#include <cassert>
+
+namespace storage {
+
+/**
+ * Returns the super bucket key of the given bucket id key based on the minimum used bits allowed.
+ */
+uint64_t get_super_bucket_key(const document::BucketId& bucket_id) {
```

Comment:
Should this be `inline` to avoid potential of causing duplicate definition errors? Could also be `noexcept`

Reply:
Fixed